### PR TITLE
Adding a flag to run_local_database.py to allow populating the databa with initial random data.

### PR DESCRIPTION
**Pull request by erzel**
@alainjobart 
@dumbunny 
Please take a look.

---
This change is [Reviewable](https://reviewable.io/reviews/youtube/vitess/1646)

### CODE REVIEW

**Comment by thompsonja** [comment]
*[py/vttest/local_database.py, line 21 \[r1\]](https://reviewable.io:443/reviews/youtube/vitess/1646#-KFpeW2u31erXNsc4DO3:-KFpeW2u31erXNsc4DO4:1006011483) ([raw file](https://github.com/youtube/vitess/blob/5d11887f455e23bd3b16e0315be3d3f0f9026faf/py/vttest/local_database.py#L21)):*
I suggest combining all of the randomized data flags into a single struct (initialize_with_random_data_options perhaps) to keep the constructor simple.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/youtube/vitess/1646)*

File: py/vttest/local_database.py [hunk 1]
```diff
@@ -11,11 +13,26 @@
 class LocalDatabase(object):
   """Set up a local Vitess database."""
 
-  def __init__(self, shards, schema_dir, vschema, mysql_only, web_dir=None):
+  def __init__(self,
```
Comment (alainjobart) [inline py/vttest/local_database.py]: please add docstring

File: py/vttest/local_database.py [hunk 2]
```diff
@@ -123,6 +154,153 @@ def load_schema(self):
...
+    return self.rng.uniform(0, 1) < true_probability
+
+  def random_integer(self, field_type, num_bytes):
+    num_bits = 8*num_bytes
+    if field_type.endswith('unsigned'):
+      return '%d' % (self.rng.randint(0, 2**num_bits-1))
+    return '%d' % (self.rng.randint(-2**(num_bits-1), 2**(num_bits-1)-1))
+
+  def random_float(self):
+    # Interpret a random 32-bit string as a float.
```
Comment (alainjobart) [inline py/vttest/local_database.py]: I think there are illegal values in there, like NAN... Maybe needs to be double-checked.

**Comment by erzel** [comment]
Review status: 0 of 3 files reviewed at latest revision, 3 unresolved discussions.

---

*[py/vttest/local_database.py, line 16 \[r1\]](https://reviewable.io:443/reviews/youtube/vitess/1646#-KFpRmYT9-MiT3p1_jYs-r1-16:-KFqYUc8D3-YYnip1GvZ:-499146009) ([raw file](https://github.com/youtube/vitess/blob/5d11887f455e23bd3b16e0315be3d3f0f9026faf/py/vttest/local_database.py#L16)):*
Done.

---

*[py/vttest/local_database.py, line 21 \[r1\]](https://reviewable.io:443/reviews/youtube/vitess/1646#-KFpeW2u31erXNsc4DO3:-KFqXIWD6uLCKqZC4SXW:-499146009) ([raw file](https://github.com/youtube/vitess/blob/5d11887f455e23bd3b16e0315be3d3f0f9026faf/py/vttest/local_database.py#L21)):*
Done.

---

*[py/vttest/local_database.py, line 293 \[r1\]](https://reviewable.io:443/reviews/youtube/vitess/1646#-KFpRmYT9-MiT3p1_jYs-r1-293:-KFqXKrgxVg4_kLX5c3R:216447099) ([raw file](https://github.com/youtube/vitess/blob/5d11887f455e23bd3b16e0315be3d3f0f9026faf/py/vttest/local_database.py#L293)):*
You're right. I was lazy. Changed. PTAL.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/youtube/vitess/1646)*

**Comment by alainjobart** [comment]
LGTM

**Pull request merged**